scss/_skeletons.scss

```diff
@@ -0,0 +1,60 @@
+// .skeleton-line      - the skeleton container
+// .skeleton-dash      - the "text" replacement
+
+.skeleton-line {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  // We use a &nbsp; to preserve
+  // the font size of the parent
+  &::before {
+    width: 0;
+    content: "\00A0";
+  }
+
+  // Animation
+  &.skeleton-glow {
+    animation: skeleton-glow 2s ease-in-out infinite;
+  }
+}
+
+.skeleton-dash {
+  width: 100%;
+  height: 1em;
+  background-color: currentColor;
+
+  + .skeleton-dash {
+    margin-left: 4px;
+  }
+}
+
+// Sizing
+.skeleton-xs {
+  > .skeleton-dash {
+    height: .6em;
+  }
+}
+
+.skeleton-sm {
+  > .skeleton-dash {
+    height: .8em;
+  }
+}
+
+.skeleton-lg {
+  > .skeleton-dash {
+    height: 1.2em;
+  }
+}
```

Comment:
Since only `.skeleton-dash` gets modified, why don't we apply modifiers on them instead of their parent?

By the way, I think they're not necessary as-is: setting `1em` as `height` on `.skeleton-dash` allows to use `font-size` on the parent, eg: 
```scss
.skeleton-lg {
  font-size: 1.2em;
}
```

This would lighten sources and selectors a bit, plus solve one of @mdo's comment ;)

Reply:
I've just pushed the change, but then I realized it's breaking the "line-height" trick. 
Should I revert the commit or forget about the "trick" ?

With the new commit
<img width="633" alt="Screenshot 2020-10-12 at 11 59 32" src="https://user-images.githubusercontent.com/385551/95733517-8ca77a80-0c82-11eb-9d90-a1c7bfb8ee83.png">

Before the commit
<img width="656" alt="Screenshot 2020-10-12 at 12 02 36" src="https://user-images.githubusercontent.com/385551/95733836-f9227980-0c82-11eb-86b3-d2128b3fb644.png">